GVFS/GVFS.FunctionalTests/Tools/GVFSServiceProcess.cs

```diff
@@ -3,15 +3,28 @@
 using System.Diagnostics;
 using System.IO;
 using System.Linq;
+using System.Runtime.InteropServices;
 using System.ServiceProcess;
 using System.Threading;
 
 namespace GVFS.FunctionalTests.Tools
 {
     public static class GVFSServiceProcess
     {
-        public const string TestServiceName = "Test.GVFS.Service";
-        private const string ServiceNameArgument = "--servicename=" + TestServiceName;
+        private static readonly string ServiceNameArgument = "--servicename=" + TestServiceName;
+
+        public static string TestServiceName
+        {
+            get
+            {
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+                {
+                    return "GVFS.Service";

```

Comment:
Do we have a testing plan for the service on macOS yet? I don't see any test changes in this PR, do we want to add them in this PR or in a future PR?

Reply:
I would love to see some tests come in at the beginning of this work.

Reply:
Existing test cases that test repo registration are run on the Mac as well. 

Couple additional tests are coming in the next PR (along with auto mount on user login). 
1. Service start and stop automatically triggers auto-remount of registered repositories
2. UT to verify GVFS.Service::HandleRequest routes repo-registration requests correctly